docs/topics/cli-run.adoc

```diff
@@ -76,7 +76,7 @@ To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler`
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/bin/windup-cli -Dwindup.decompiler=procyon \
+$ <{ProductShortName}_HOME>/bin/{mta-cli} -Dwindup.decompiler=procyon \
```

Comment:
The -D option doesn't exist anymore in mta-cli

Reply:
@abrugaro  what should this line look like? 
1.  $ <{ProductShortName}_HOME>/bin/{mta-cli} -windup.decompiler=procyon \
2.  $ <{ProductShortName}_HOME>/bin/{mta-cli}  \
3.  - remove this line completely?

Reply:
We should eliminate the entire section that refers to overriding the default decompiler